lib/pacemaker/pcmk_injections.c

```diff
@@ -93,7 +93,7 @@ inject_transient_attr(pcmk__output_t *out, xmlNode *cib_node,
 void
 pcmk__inject_failcount(pcmk__output_t *out, cib_t *cib_conn, xmlNode *cib_node,
                        const char *resource, const char *task,
-                       guint interval_ms, int exit_status)
+                       guint interval_ms, int exit_status, const char *offset)
```

Comment:
> Add to the doxygen comment above as well

Ah, missed it.

> 
> I'm thinking a `bool` would be clearer since we don't actually use the value

I was thinking it was kind of straight-forward to pass the value of `failed_start_offset` or `failed_stop_offset` to be used rather than parsing them outside of pcmk__inject_failcount().

Reply:
I'd be fine with that, as long as the doxygen makes clear that the only thing that makes any difference is whether it's "INFINITY" or not